repo/packages/K/kong/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "3.0",
+  "name": "kong",
+  "version": "0.10.3",
+  "scm": "https://github.com/Mashape/kong",
+  "maintainer": "https://getkong.org/community/",
+  "website": "https://getkong.org/",
+  "description": "Kong is open-source API Gateway and Microservices Management Layer, delivering high performance and reliability.",
+  "framework": false,
+  "preInstallNotes": "You need to have Kong supported DB and Marathon-LB instances running",
```

Comment:
Add this line at the beginning of `preInstallNotes`: `This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Preview packages should never be used in production!`